### DISCUSSION TITLE
How to disable the touch two finger rotation gesture?

### DISCUSSION THREAD

**Discussion by tunttunen** [jacomyal/sigma.js#1453] [Q&A] [answered]
Hi there,

Is there a native way in SigmaJS to only disable the two finger rotation gesture when using touch devices?

**Comment by tunttunen** [comment]
I guess it's done by setting the camera angle to be 0 in the touch move handler when 2 touches are in action? There's no direct "disable rotation" other than this?

**Comment by sim51** [comment, marked as answer]
You can disable the rotation  : ` sigma.setSetting("enableCameraRotation", false);` 
It has been added recently : https://github.com/jacomyal/sigma.js/pull/1449
